frontend/src/main.js

```diff
@@ -23,22 +23,15 @@ Vue.component("vue-json-pretty", VueJsonPretty);
 Vue.use(require("vue-moment"));
 Vue.use(SortUtil);
 
-var apiBaseUrl;
-var socketApi;
+var apiBaseUrl = process.env.VUE_APP_API_BASE_URL;
+var eventsHost = process.env.VUE_APP_EVENTS_HOST ? process.env.VUE_APP_EVENTS_HOST : window.location.host;
```

Comment:
Where is `VUE_APP_EVENTS_HOST` defined? I do not see it in the .env file.

Reply:
That value is not needed for normal production configuration. That would be used if the API/backend and frontend are on different hosts. For instance, in my `env.development.local` I have:
```
VUE_APP_API_BASE_URL=http://localhost:48080/api
VUE_APP_EVENTS_HOST=localhost:48080
VUE_APP_EVENTS_PATH=events
```
Where my frontend is running at localhost:8080 and my backend is localhost:48080.